torchmultimodal/utils/common.py

```diff
@@ -96,18 +97,14 @@ def tensor_slice(x: Tensor, begin: List[int], size: List[int]) -> Tensor:
 
 
 def load_module_from_url(
-    model: torch.nn.Module, url: str, progress: bool = True
+    model: torch.nn.Module, url: str, strict: bool = True, progress: bool = True
```

Comment:
nit: pass device?

Reply:
so device is defaulting to wherever ckpt was saved or cpu if gpu not available.